models/health.go

```diff
@@ -89,6 +94,7 @@ type ProxyStatus struct {
 type RequestHealth struct {
 	Inbound  map[string]map[string]float64 `json:"inbound"`
 	Outbound map[string]map[string]float64 `json:"outbound"`
+	Config   string                        `json:"config"`
```

Comment:
This should be `omitempty` as this is optional and only used on the case where user annotates a specific Service and/or Workload.

Reply:
I'm reviewing the appender and looking that code I guess we should call this parameter "healthConfig", also here and in the graph to get consistency.
Rationale is that /health endpoint and /graph fetch the info needed to calculate health: rate request, if "healthConfig" is present in both, then instead to use the global "healthConfig" config it personalizes for this item (service/workload) using this info.